rest-service/manager_rest/test/endpoints/test_plugins_updates.py

```diff
@@ -118,12 +118,11 @@ def test_raises_with_nonexisting_blueprint(self):
             self.client.plugins_update.update_plugins("non_existing")
             self.assertEqual(404, e.exception.status_code)
 
-    def test_raises_blueprint_has_no_deployments(self):
+    def test_no_op_blueprint_has_no_deployments(self):
```

Comment:
not clear of a name
=> test_basic_noting_to_plugin_update

Reply:
Changed to `test_plugins_update_does_nothing_when_no_deployments_to_update`